### PR TITLE
[MI-2712]: Create MultiProgress component

### PR DESCRIPTION
**Pull request by SaurabhSharma-884**
### Ticket ID:
https://brightscout.atlassian.net/browse/MI-2712

### Description:
- Add MultiProgress component

### Default
Screenshot from 2023-02-06 18-23-09

### CODE REVIEW

**Review by atisheyJain03 — CHANGES_REQUESTED** [review]
.

**Review by atisheyJain03 — CHANGES_REQUESTED** [review]
.

File: src/components/MultiProgress/MultiProgress.component.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ * ```
+ *
+ */
+export const MultiProgress = (props: MultiProgressBarProps) => {
+    const {progressBarConfig, className} = props;
+
+    // get percentage of progress of a progress bar with respect to progress of all the progress bars
+    const getBarFraction = (value: number) => {
+        let total = 0;
+        progressBarConfig.forEach((bar: ProgressBarProps) => {
```
Comment (atisheyJain03) [inline src/components/MultiProgress/MultiProgress.component.tsx:26]: can we save this in a constant to save recomputation every time?

File: src/components/MultiProgress/MultiProgress.component.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        });
+        return (value / total) * 100;
+    };
+
+    return (
+        <StyledMultiProgressBarContainer>
+            <ProgressBar className={`mm-multiProgress ${className}`}>
+                {progressBarConfig.map((bar: ProgressBarProps, index: number) => (
+                    <ProgressBar
+                        key={index}
```
Comment (atisheyJain03) [inline src/components/MultiProgress/MultiProgress.component.tsx:37]: index as key is not a good option for this. Instead, send the key in bar object

Comment (atisheyJain03) [inline src/components/MultiProgress/MultiProgress.component.tsx:37]: index as key is not a good option for this. Instead, send the key in bar object

**Review by atisheyJain03 — CHANGES_REQUESTED** [review]
Good work

File: src/components/MultiProgress/MultiProgress.component.tsx (hunk 1 above)
Comment (SaurabhSharma-884) [reply src/components/MultiProgress/MultiProgress.component.tsx:26]: Modified

**Comment by SaurabhSharma-884** [comment]
The work of this PR was already present in https://github.com/Brightscout/mm-ui-library/pull/61 and is merged. Hence, Closing this PR

**Pull request closed without merging**